flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScanSql.java

```diff
@@ -178,6 +179,38 @@ public void testInferedParallelism() throws IOException {
     Assert.assertEquals("Should produce the expected parallelism.", 1, parallelism);
   }
 
+  @Test
+  public void testExposeLocality() throws Exception {
```

Comment:
It only verifies data read. this doesn't really verify locality aware assignment. Ideally, we need 2 files stored in 2 hosts with HDFS and run a cluster of TMs on those two hosts. Then we can verify the assigned files/splits are from the same host. But I am not sure if this can be done in a unit test setup.

Reply:
Yes, +1, ideally it is, but I haven't found a way to achieve it.So here only test whether it works properly when `table.exec.iceberg.expose-split-locality-info` is set to false.

Reply:
@hililiwei How does Flink code base test this feature?

Reply:
Seems to be tested by manually specifying the hostname. more refer: https://github.com/apache/flink/blob/master/flink-core/src/test/java/org/apache/flink/core/io/LocatableSplitAssignerTest.java
Or try to test it by introducing miniDFS, but the project doesn't seem willing to introduce it.

Reply:
this is for unit test the assigner, not an e2e test of the whole thing

Reply:
Yes, during the test phase, I printed some logs to see if it was working properly, such as this one:

![2022-1-17-2](https://user-images.githubusercontent.com/59213263/149700947-bea570d1-8cb3-4c16-9dbb-e690eb69535d.PNG)

![2022-1-17](https://user-images.githubusercontent.com/59213263/149699989-0dfb1440-0896-407f-9c10-775d887e6179.PNG)